chat-commands.js

```diff
@@ -1409,6 +1410,37 @@ exports.commands = {
 		user.leaveRoom(targetRoom, connection);
 	},
 
+	getstaff: function (target, room, user) {
+		if (!target) return this.parse('/help getstaff');
+		const staff = Rooms('staff');
+		if (!staff) return this.errorReply("This server does not have a staff room, therefore there is no room to notify of your request.");
+		target = target.trim();
+		const separator = 'answer request id ';
+		if (target.startsWith(separator)) {
+			if (!this.can('lock')) return;
+			const requestNumber = target.substr(separator.length, target.length);
+			if (!staff.requests || !staff.requests[requestNumber]) return this.errorReply("Werid, this request doesn't seem to exist.");
+			const request = staff.requests[requestNumber];
+			user.joinRoom(request.roomId);
+			staff.add(Chat.html`|uhtmlchange|staff-request-${requestNumber}|<div class="infobox">[Help Request] <button class="button disabled" style="font-weight: bold; color: #575757; font-weight: bold; background-color: #d3d3d3">${request.roomId} (${user.name} answered)</button> - ${request.by} said: ${request.message}</div>`).update();
+		} else {
+			if (!user.trusted) return this.errorReply("Only trusted users are allowed to use this command. If you need help or a global staff member, ask the Help room.");
+			if (user.lastStaffRequest &&  Date.now() - user.lastStaffRequest < STAFF_REQUEST_THROTTLE) return this.errorReply("You cannot submit another global staff request so soon.");
```

Comment:
Shouldn't this be a per-room basis - better if 4 trusted users don't use it at the same time unknowingly?

Reply:
Good idea, this is done